quickstart.mdx

```diff
@@ -3,391 +3,133 @@ title: "Quickstart"
-
-First, import PromptLayer and create a PromptLayer client. We'll use this client to call OpenAI.
-
-Behind the scenes, the PromptLayer library is a wrapper around OpenAI's SDK. This means that all LLM requests are still made locally from your machine. PromptLayer functions as a sidecar, logging the response after the request is made.
-
-<CodeGroup>
-
-```python Python openai >= 1.0.0
-# Make sure to `pip install promptlayer`
-import os
-os.environ["OPENAI_API_KEY"] = "sk-<your_openai_api_key>"
-
-from promptlayer import PromptLayer
-promptlayer_client = PromptLayer(api_key="<your_promptlayer_api_key>")
-
-# Swap out your 'from openai import OpenAI'
-OpenAI = promptlayer_client.openai.OpenAI
-client = OpenAI()
-```
-
-
-```js JavaScript
-// Make sure to `npm install promptlayer`
-import { PromptLayer } from "promptlayer";
-const promptLayerClient = new PromptLayer();
-
-// Make sure you have openai installed with `npm install openai`
-const OpenAI = promptLayerClient.OpenAI;
-const openai = new OpenAI();
-```
-
-</CodeGroup>
-    
-## Retrieve & Run the Prompt
+## Creating a Prompt
 
-Remember the `{topic}` user input variable we made to specify the haiku topic in our prompt? PromptLayer can format the f-string or Jinja2 template using this variable.
-
-We will use PromptLayer to fetch the latest version of the prompt, inject input variables, and run it.
-
-<CodeGroup>
-
-```python Python
-input_variables = {
-    "topic": "The American Revolution"
-}
-
-response = promptlayer_client.run(
-    prompt_name="ai-poet", input_variables=input_variables)
-
-# Using OpenAI format
-print(response["raw_response"].choices[0].message.content)
-```
-
-```js JavaScript
-const input_variables = {
-  topic: "The American Revolution"
-};
-
-const response = await promptLayerClient.run({
-  promptName: "ai-poet", 
-  inputVariables: input_variables,
-});
-
-// Using OpenAI format
-console.log(response["raw_response"].choices[0].message.content);
-```
-
-</CodeGroup>
-
-<Tip>
-  For simplicity, the code snippet above will work with OpenAI only. Use the [prompt blueprint](/quickstart-part-two#prompt-blueprint) return item for a model-agnostic response format. 
-  ```python
-  print(response["prompt_blueprint"]["prompt_template"]["messages"][-1]["content"])
-  ```
-</Tip>
-
-Amazing! `promptlayer_client.run` will automatically format your prompt template into the format needed by OpenAI, Anthropic, and all other major model providers.
-
-The LLM request runs locally on your machine.
+1. Navigate to the prompt registry from PromptLayer's home screen
+2. Click "Create Prompt" and provide a name and description
+3. Give your template a title in the 'Title' field
+4. In the "SYSTEM" field, paste your system message
+5. Add a new message with input variables using curly braces, like `{variable_name}`
+6. Choose a model by clicking "Parameters"
+7. Save the prompt template
```

Comment:
This is super confusing and feels half-baked.

Why are we not using a real example like the ai-poet from before?

Why are there no pictures or gifs?